apps/cli/taboot_cli/commands/ingest_github.py

```
@@ -0,0 +1,203 @@
+
+    Raises:
+        typer.Exit: Exit with code 1 if ingestion fails.
+    """
+    try:
+        # Validate repo format
+        if "/" not in repo or repo.count("/") != 1:
+            console.print(
+                f"[red]✗ Invalid repository format: {repo}[/red]\n"
+                f"[yellow]Expected format: 'owner/repo' (e.g., 'anthropics/claude')[/yellow]"
+            )
+            raise typer.Exit(code=1)
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)